src/qoa.js

```diff
@@ -62,6 +62,10 @@ class Qoa {
     return new Secure(this._buildConfig(x)).request();
   }
 
+  clearScreen() {
+    return process.stdout.write('\u001Bc');
+  }
+
```

Comment:
I can see how this could be useful, though it would be even better to perform the console clearing operation entirely through the [`tty.WriteStream`](https://nodejs.org/api/tty.html#tty_class_tty_writestream) API:

- by initially moving the cursor to `(0, 0)`, top-right corner, of the console with the [`writeStream.cursorTo(x, y)`](https://nodejs.org/api/tty.html#tty_writestream_cursorto_x_y) function
- and then use the [`writeStream.clearScreenDown()`](https://nodejs.org/api/tty.html#tty_writestream_clearscreendown) function to clear the WriteStream from the cursor down

```js
  clearScreen() {
    process.stdout.cursorTo(0, 0);
    process.stdout.clearScreenDown();
  }
```

Reply:
Yes, I notice that, the way I preferred can be not work on other OSes, also there is console.clear() function, What do you think about that? Which one is good?

Reply:
It is available only on node [`=> v8.3.0`](https://nodejs.org/api/console.html#console_console_clear), thus would make the package unsusable for node `< v8.3.0` users, ie node [`v6.x`](https://nodejs.org/dist/latest-v6.x/).